test_config/motor_test/jaws.cmd

```diff
@@ -0,0 +1,3 @@
+$(IFIOC_GALIL_01=#) dbLoadRecords("$(JAWS)/db/split_jaws_header_vertical.db","P=$(MYPVPREFIX)MOT:,JAWS=JAWS1:,mXN=MTR0101,mXS=MTR0102,mXW=MTR0202,mXE=MTR0201")
```

Comment:
If you look at how we test the `motorExtensions` we put the test settings files into the `motorExtensions` directory, I think putting this in the `jaws` directory might be better.

Reply:
Done. Added folders for each type of jawset in `jaws\settings\` containing a `.cmd` each and updated the path in `vertical_jaws.py` IOC tests to match.